### ISSUE TITLE
Empty required_css_class not handles well

### ISSUE THREAD

**Issue by Suor** [tutorcruncher/django-jinja-bootstrap-form#2]
Here is what I get for default login form:

``` html
      <label class="control-label  {{ no such element: django.contrib.auth.forms.AuthenticationForm object['required_css_class'] }}" for="id_username">Username</label>
```

**Comment by samuelcolvin** [comment]
ye it's a bug that could do with fixing but I think it will only appear with `DEBUG = True`, it should be blank in production.

**Comment by samuelcolvin** [comment]
Fixed with [v4.1.0](https://github.com/samuelcolvin/django-jinja-bootstrap-form/releases/tag/v4.1.0)
